src/sidebar/components/MarkdownEditor.js

```diff
@@ -97,6 +98,31 @@ function handleToolbarCommand(command, inputEl) {
   }
 }
 
+/**
+ * @typedef {import("@hypothesis/frontend-shared/lib/components/Link").LinkProps} LinkProps
+ *
+ * Style a Link to look like an IconButton, including touch sizing
+ * affordances.
+ *
+ * @param {Omit<LinkProps, 'children'> & { icon: string }} props
+ */
+function IconLink({ classes, icon, linkRef, ...restProps }) {
```

Comment:
Given that this really needs to match the look of a `ToolbarButton`, I wonder if it would make sense to call it a `ToolbarLink`. I was going to suggest integrating it into `ToolbarButton`, but the props are going to be different (eg. no `onClick` handler for a link.

Reply:
The structure here is:

* `IconLink` -> "extends" `Link` to be styled like an `IconButton`
* `Toolbar` -> passes additional `classes` to `IconLink` to "extend" it to look like other `ToolbarButton`s, which are not square. This happens here: https://github.com/hypothesis/client/pull/4390/files#diff-84754c75093586f2425564752b5ca35ec1abcd241f840afaa2876229bacb95b1R424-R428

Thus, `IconLink` doesn't match `ToolbarButton` without those additional styles, leaving me thinking that the naming is appropriate as-is...does that make sense?